content/docs/user-guide/interactivity/robotics/concepts-and-components-overview.md

```diff
@@ -0,0 +1,107 @@
+
+If you intend to add your own sensor, it might be useful to look at how sensors already provided within the O3DE ROS2 Gem are implemented.
+
+### Robot Control
+
+The Gem comes with `ROS2RobotControlComponent`, which you can use to move your robot through:
+
+- [Twist](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Twist.msg) messages.
+- [AckermannDrive](https://github.com/ros-drivers/ackermann_msgs/blob/master/msg/AckermannDrive.msg)
+  The component subscribes to these command messages on a configured topic. The topic is `cmd_vel` by default, in a namespace as dictated by __ROS2Frame__.
+
+To make use of received command messages, use either `AckermannControlComponent`, `RigidBodyTwistControlComponent`, or `SkidSteeringControlComponent`, depending on the steering type. You can also implement your own control component or use LUA scripting to handle these commands. Unless scripting is used, control components should translate ROS 2 commands to events on `VehicleInputControlBus`. These events will be handled by a [`VehicleModelComponent`](#vehicle-model) if it is present. You can use tools such as [rqt_robot_steering](https://index.ros.org/p/rqt_robot_steering/) to move your robot with Twist messages. `RobotControl` is suitable to use with [ROS 2 navigation stack](https://navigation.ros.org/). It is possible to implement your own control mechanisms with this component.
+
+### Vehicle Model
+
+`VehicleModelComponent` serves the purpose of converting inputs such as target velocity, steering or acceleration to physical forces on parts of a vehicle (robot). `VehicleModel` has a `VehicleConfiguration` which is used to define axles, parametrize and assign wheels. The model requires a `WheelControllerComponent` present in each wheel entity. It also uses an implementation of `DriveModel`, which converts vehicle inputs to forces acting on steering elements and wheels.
+
+### Vehicle Dynamics
+
+See the [Vehicle Dynamics](vehicle-dynamics.md) section.
+
+### Spawner
+
+`ROS2SpawnerComponent` handles spawning entities during simulation. Available spawnables have to be set up as the component's field before the simulation. User is able to define named spawn points inside the Editor. This can be done by adding `ROS2SpawnPointComponent` to a child entity of an entity with `ROS2SpawnerComponent`. During the simulation user can access names of available spawnables and request spawning using ros2 services. The names of services are `/get_available_spawnable_names` and `/spawn_entity` respectivly. GetWorldProperties.srv and SpawnEntity.srv types are used to handle these features. In order to request defined spawn points names user can use `/get_spawn_points_names` service with `GetWorldProperties.srv` type. Detailed information about specific spawn point (e.g. pose) can be accessed using `/get_spawn_point_info` service with `GetModelState.srv` type. All used services types are defined in gazebo_msgs package.
```

Comment:
In the future, it would help the user to have this information separated out into a focused section. What I mean is, this document is focused on providing the user with an overview of ROS 2 concepts and understanding how things work together -- it's all conceptual. In another document, you want to focus on "how to do xyz" and only that. (XYZ can be something general but specific, like "Setting up and using spawners")

It's usually not necessary to go into all of this detail at the high-level because it's easy for the details to get lost. 

```suggestion
`ROS2SpawnerComponent` handles spawning entities during a simulation. Before the simulation, you must set up the available spawnables and define the named spawn points in the component's properties via the **O3DE Editor**. This can be done by adding `ROS2SpawnPointComponent` to a child entity of an entity with `ROS2SpawnerComponent`. During the simulation, you can access the names of available spawnables and request spawning by using ROS 2 services. The names of services are `/get_available_spawnable_names` and `/spawn_entity` respectivly. GetWorldProperties.srv and SpawnEntity.srv types are used to handle these features. In order to request the defined spawn point names, you can use the `/get_spawn_points_names` service with the `GetWorldProperties.srv` type. Detailed information about specific spawn point, such as pose, can be accessed using the `/get_spawn_point_info` service with the `GetModelState.srv` type. All used services types are defined in the **gazebo_msgs** package.
```